core/app/models/spree/order_update_attributes.rb

```diff
@@ -0,0 +1,42 @@
+module Spree
+  class OrderUpdateAttributes
+    # @param order [Order] existing (persisted) order
+    # @param attributes [Hash] attributes which are assigned to the new order.
+    #   These attributes should already have been filtered.
+    #   * :payments_attributes attributes
+    def initialize(order, attributes, request_env: nil)
```

Comment:
@jhawthorn can we add `current_user: nil` to this?  (And supply it when we call this). It would be really nice for logging and auditing to have that.